codice/ddf-ui
# Add browser incompatible page for IE

Displays a browser incompatible page when users try to access the system from an incompatible browser.

Supported browsers:
- Microsoft Edge, versions 80+
- Mozilla Firefox, versions 72+
- Google Chrome, versions 80+

(Supported versions were chosen by finding the version released closest to the start of 2020)

Unsupported browsers:
- Internet Explorer
- by default, anything else

![image](https://user-images.githubusercontent.com/6109231/96518614-42516b00-1239-11eb-815c-01ba87c62de0.png)
Regular size browser

![image](https://user-images.githubusercontent.com/6109231/96518662-5d23df80-1239-11eb-93b9-3cd522001001.png)
Small browser

Testing Instructions:
1. Build and run DDF.
1. Build this PR and follow the instructions in https://github.com/codice/ddf-ui/blob/master/README.md to install the catalog-ui.
1. Open in Internet Explorer.
1. Verify that the above page displays instead of the usual UI.
1. Repeat with all unsupported versions.
1. Verify that the usual UI displays in supported versions.
